src/thinblock.cpp

```diff
@@ -1101,6 +1101,36 @@ void SendXThinBlock(CBlock &block, CNode *pfrom, const CInv &inv)
     pfrom->blocksSent += 1;
 }
 
+bool IsThinBlockValid(const CNode* pfrom, const std::vector<CTransaction>& vMissingTx, const CBlockHeader& header)
+{
+    // Check that that there is at least one txn in the xthin and that the first txn is the coinbase
```

Comment:
it's fine, it returns if empty..., but i'll change it to .empty() rather than checking for size 0...that will be more clear.